src/api/routes/songs.ts

```diff
@@ -6,7 +6,26 @@ const router = express.Router();
 // GET request /songs
 router.get("/", (req: Request, res: Response, next: NextFunction) => {
     try {
-        res.status(200).json(data);
+        if (req.query) {
+            const artist:any = req.query.artist;
+            const genre:any = req.query.genre;
+            const language:any = req.query.language;
```

Comment:
The type-annotations must have a preceding space.
```ts
  const artist: any = req.query.artist;
  const genre: any = req.query.genre;
  const language: any = req.query.language;
```

Reply:
Done!